lib/HTFeed/Log/Layout/PrettyPrint.pm

```diff
@@ -28,17 +28,19 @@ sub new {
 }
 
 sub render {
-    my($self, $message, $category, $priority, $caller_level) = @_;
+    my($self, $incoming_message, $category, $priority, $caller_level) = @_;
```

Comment:
Changes here are because previously this method mutated the incoming message; now it duplicates it so that mutation doesn't affect the message for other appenders that might be in use